source/draft_api.rst

```diff
@@ -600,7 +650,17 @@ Response body:
        "created_at":"2014-04-17",
        "updated_at":null,
        "deleted_at":null,
-       "revision":1
+       "revision":1,
+       "users": {
+         "members": [
+           "patcht"
+         ],
+         "spectators": [
+         ],
+         "managers": [
+           "tschuy"
+         ]
+       }
     }
 
 Note that this endpoint, when called, will automatically set the currently
```

Comment:
If this is still true, shouldn't there be someone in the spectator's field? If the user ``tschuy`` created the project he should be in spectators and members as well.